apps/condo/domains/user/schema/OidcClient.js

```diff
@@ -74,7 +74,7 @@ const OidcClient = new GQLListSchema('OidcClient', {
         },
 
         expiresAt: {
-            schemaDoc: 'The timestamp of the token expiration',
+            schemaDoc: 'The timestamp of the client expiration',
```

Comment:
Please run `maketypes` script to become graphqlschema up to date

Reply:
Done :)
Thanks!